dll/win32/kernel32/kernel32.spec

```diff
@@ -148,7 +148,7 @@
 @ stdcall CreateMutexW(ptr long wstr)
 @ stdcall CreateNamedPipeA(str long long long long long long ptr)
 @ stdcall CreateNamedPipeW(wstr long long long long long long ptr)
-@ stdcall -version=0x501-0x502 CreateNlsSecurityDescriptor(ptr long long)
```

Comment:
Which Windows version have you checked? Because on 2003 the function has 3 parameters.